java-util/pom.xml

```diff
@@ -85,6 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.datasketches</groupId>
```

Comment:
I haven't agreed.  And @leventov  and I are busy working out a solution that should work for both DataSketches and Druid.  Let's not jump to conclusions before we determine that a fork is the only solution.